packages/statemanager/package.json

```diff
@@ -52,6 +52,7 @@
     "@ethereumjs/rlp": "^5.0.1",
     "@ethereumjs/trie": "^6.0.1",
     "@ethereumjs/util": "^9.0.1",
+    "@ethereumjs/verkle": "^0.0.1",
```

Comment:
Hmm, maybe also for the protocol: now we have the verkle package hardcoded and included into production code. Is this something we can (want to) live with? 🤔 Maybe we should give potential side effects some thought.

Respectively when thinking about it:

What we *DO* side-add here is an additional 500KB+ weight for the VM (in particular, SM of course as well) on dependencies through the current `rust-verkle-wasm` dependency (`577077 12 Nov 16:14 rust_verkle_wasm_bg.wasm`).

I think we really *NOT* want to do that, now thinking about that twice. 🤔 Might (will likely) be necessary at some point down the line, but for now it's far far too early to justify this addition for the production users (and maybe up till then we might be able to further optimize on the wasm library or ship with JS).

So I think we need to come up with a bit more manual solution here and pushing this to the optional peer dependencies for now.

Reply:
Yeah exactly, it's used for pedersenhash. Will update to peerDependency in stateManager and normal dep in the client.

Reply:
pushed